docs/how-to/import-prisma-schema.md

```diff
@@ -0,0 +1,120 @@
+- Map indexes in the database to indexes in the Prisma schema  
+- Map database constraints to attributes or type modifiers in the Prisma schema
+
+To learn more see [Prisma's Introspection docs](https://www.prisma.io/docs/concepts/components/introspection#what-does-introspection-do).
+
+Run the `prisma db pull` command in the terminal to start the introspection process.
+Using this command requires your database's **Connection URL** be set in your Prisma Schema `datasource`.
+
+```bash
+npx prisma db pull
+```
+
+:::caution
+After generating your schema file, double check it in your code editor for errors.
+Make sure you fix any errors before you upload your `schema.prisma` file into Amplication.
+There are some cases where the introspection process ends with a schema that might have errors. 
+:::
+
+Here's an example of where the generated schema file could have an error:
+
+```graphql title="schema.prisma"
+model searchText {
+  application_id   BigInt  @id @default(0) @db.UnsignedBigInt
+  guid             String  @db.VarChar(50)  
+  application_text String? @db.LongText   
+
+  @@index([application_text], map: "idx_searchText_application_text")
+}
+```
+
+In this case, it's because you cannot define an index on fields with a native type of **LongText** on MySQL. You have to use the `length` argument to the field in the index definition to allow this.
+
+To learn more about other prisma schema issues you might run into, see [Prisma's Error Message Reference](https://www.prisma.io/docs/reference/api-reference/error-reference).
+
+### Step 2: Upload your Prisma Schema File to Amplication
+
+First, visit the _Entities_ page of your service.
+
+![Upload Prisma Schema Button](./assets/upload-prisma-schema-button.png)
+
+If you're subscribed to a Pro or Enterprise plan, you'll see the **Upload Prisma Schema** button in the top right corner.
+
+Click on this button and you'll be presented a form where you can upload your `schema.prisma` file.
+
+![Upload Prisma Schema](./assets/upload-prisma-schema.png)
+
+### Step 3: Check the conversion logs
+
+After uploading the file, you'll see a log of what's happening while Amplication is [converting your prisma schema](/getting-started/how-prisma-schema-is-converted-into-entities) into its internal structure.  
+
+Here's how that log might look:
+
+```bash title="Prisma Schema Conversion Log" 
+2023-07-06T09:47:39.410Z (Warning) The option 'PENDING' has been created in the enum 'OrderStatus', but its value has not been mapped
+2023-07-06T09:47:39.410Z (Info) The option 'COMPLETED' has been created in the enum 'OrderStatus'
+2023-07-06T09:47:39.410Z (Info) The option 'CANCELLED' has been created in the enum 'OrderStatus'
+2023-07-06T09:47:39.411Z (Warning) The option 'INDIVIDUAL' has been created in the enum 'CustomerType', but its value has not been mapped  
+2023-07-06T09:47:39.411Z (Warning) The option 'COMPANY' has been created in the enum 'CustomerType', but its value has not been mapped
```

Comment:
use a screenshot, current example is confusing since it does not looks like what the user sees